book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/repository/IEventRepository.java

```diff
@@ -10,8 +10,8 @@
 import java.util.UUID;
 
 @Repository
-public interface EventRepository extends JpaRepository<Event, UUID> {
+public interface IEventRepository extends JpaRepository<Event, UUID> {
 
-    @Query("SELECT e FROM Event e JOIN FETCH e.vendor JOIN FETCH e.venue JOIN FETCH e.eventType")
+    @Query("SELECT e FROM Event e JOIN FETCH e.vendor JOIN FETCH e.venue JOIN FETCH e.eventType ORDER BY e.startDate DESC")
```

Comment:
This query is still not required. Just passing the pageable will do the job. @ManyToOne annotation are FetchType.Eager by default hence JPA will use joins to fetch data.